docs/source/how-to/understand-keys-and-how-to-change-them.md

```diff
@@ -0,0 +1,146 @@
+---
+jupytext:
+  text_representation:
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
+# Understanding Intake-ESM Keys and How to Use Them
```

Comment:
I'm in the process of only capitalizing the first word in the title and leaving the rest in lower case... 
```suggestion
# Understanding intake-ESM keys and how to use them
```